bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1493,21 +1515,27 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 
 
 <!-- clm5_1 and cam7.0 -->
-<!-- NB: This pSASU finidat is from an 1850 spin-up -->
 <finidat hgrid="0.9x1.25"    maxpft="79"  mask="gx1v7" use_cn=".true." use_cndv=".false." use_fates=".false."
-         ic_ymd="20000101" sim_year="2000" do_transient_pfts=".false." use_excess_ice=".true."
+         ic_ymd="20000101" sim_year="2000" use_excess_ice=".true."
          ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".true."
          phys="clm5_1" use_init_interp=".true."
->lnd/clm2/initdata_esmf/ctsm5.3/ctsm52026_f09_pSASU.clm2.r.0421-01-01-00000.nc
+>lnd/clm2/initdata_esmf/ctsm5.3/ctsm53n04ctsm52028_f09_g17_BgcCrop_exice_hist.clm60.r.2000-01-01.nc
 </finidat>
 
 <!-- clm6_0 and cam7.0 -->
-<!-- NB: This pSASU finidat is from an 1850 spin-up -->
+<!-- slevis-lmwg, ekluzek 2024/10/11: do_transient_pfts was removed here to bypass a bug that doesn't allow
```

Comment:
@slevis-lmwg and I figured out that this is a bug when finidat files were made with a transient case, but used for a control case. We'd like this to work correctly so that do_transient_pfts=.true. and use_init_interp could also be removed. It works properly the opposite way, but not this way.

Reply:
Added a note on this to #2169